app/src/main/java/com/worldtreeinc/leaves/activity/BidderDashActivity.java

```diff
@@ -61,28 +59,21 @@ private void initialize(){
 
     @Override
     public boolean onCreateOptionsMenu(Menu menu) {
-        // Inflate the menu; this adds items to the action bar if it is present.
         getMenuInflater().inflate(R.menu.menu_bidder_dash, menu);
         return true;
     }
 
     @Override
     public boolean onOptionsItemSelected(MenuItem item) {
-        // Handle action bar item clicks here. The action bar will
-        // automatically handle clicks on the Home/Up button, so long
-        // as you specify a parent activity in AndroidManifest.xml.
         int id = item.getItemId();
 
-        //noinspection SimplifiableIfStatement
         if (id == R.id.action_settings) {
             return true;
         }else if(id == R.id.action_logout){
```

Comment:
NITPICK - You should not have the else part of the if statement since it is preceded by return statement. Remember that the return statement itself breaks the program flow.